frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -52,86 +51,112 @@ const getIconClass = (status: ClusterUpdateStatus) => {
-  View <Link to="/settings/cluster/clusteroperators?orderBy=desc&sortBy=Status">Cluster Operators</Link> for more details
-  {updatesAvailable && ' or check other available updates to try another version'}.
-</div>;
+const UpdatesAvailableMessage: React.SFC<CVStatusMessageProps> = ({cv, iconClass}) => <React.Fragment>
+  <div className="co-update-status">
+    {iconClass && <i className={classNames(iconClass, 'update-pending')} aria-hidden={true}></i>}
+    &nbsp;
+    Update available
+  </div>
+  <div>
+    <Button bsStyle="primary" onClick={() => (clusterUpdateModal({cv}))}>
+      Update now
+    </Button>
+  </div>
+</React.Fragment>;
 
-const RetrieveUpdatesFailedAlert: React.SFC<RetrieveUpdatesFailedAlertProps> = ({condition}) => <div className="alert alert-danger">
-  <i className="pficon pficon-error-circle-o" aria-hidden="true" />
-  <strong>Could not retrieve updates.</strong>
-  {condition.message && ` ${condition.message}.`}
-</div>;
+const UpdatingMessage: React.SFC<CVStatusMessageProps> = ({cv, iconClass}) => {
+  const updatingCondition = getClusterVersionCondition(cv, ClusterVersionConditionType.Progressing, K8sResourceConditionStatus.True);
+  return <React.Fragment>
+    <div>
+      {iconClass && <i className={iconClass} aria-hidden={true}></i>}
+      &nbsp;
+      Working towards
+    </div>
+    <div>updatingCondition.message && `${updatingCondition.message}. `</div>
+    <Link to="/settings/cluster/clusteroperators?orderBy=desc&sortBy=Status">View detailed progress</Link>
+  </React.Fragment>;
+};
 
-const UpdateInProgressAlert: React.SFC<UpdateInProgressAlertProps> = ({condition}) => <div className="alert alert-info">
-  <i className="pficon pficon-info" aria-hidden={true} />
-  <strong>Cluster update in progress.</strong>
+const ErrorRetrievingMessage: React.SFC<StatusMessageProps> = ({iconClass}) => <div>
+  {iconClass && <i className={iconClass} aria-hidden={true}></i>}
   &nbsp;
-  {condition.message && `${condition.message}. `}
-  View <Link to="/settings/cluster/clusteroperators?orderBy=desc&sortBy=Status">Cluster Operators</Link> for more details.
+  Error retrieving
 </div>;
 
-const UpdatesAvailableAlert: React.SFC<UpdatesAvailableAlertProps> = ({cv}) => {
-  const currentlyUpdating = isProgressing(cv) || updateFailing(cv);
-  const titleText = currentlyUpdating ? 'Other updates are available.' : 'Cluster update is available.';
-  const buttonText = currentlyUpdating ? 'Update to a different version.' : 'Update now.';
-  return <div className="alert alert-info">
-    <i className="pficon pficon-info" aria-hidden={true} />
-    <strong>
-      {titleText}
-    </strong>
-    <Button bsStyle="link" className="co-m-modal-link" onClick={()=> (clusterUpdateModal({cv}))}>
-      {buttonText}
-    </Button>
-  </div>;
-};
+const FailingMessage: React.SFC<StatusMessageProps> = ({iconClass}) => <React.Fragment>
+  <div>
+    {iconClass && <i className={iconClass} aria-hidden={true}></i>}
+    &nbsp;
```

Comment:
I would put this on one line so you can avoid the `&nbsp`. Better not to output that if `iconClass` isn't defined. (Here and below.)